src/AspNetCore/Common/GraphQLQuery.cs

```diff
@@ -22,12 +22,7 @@ public Inputs GetInputs()
 
         public static Inputs GetInputs(JObject variables)
         {
-            if (variables == null)
-                return null;
-
-            var inputs = variables.GetValue() as Dictionary<string, object>;
-
-            return new Inputs(inputs);
+            return variables.ToInputs();
```

Comment:
Should this perhaps be removed entirely?

Reply:
Yes. Just using one would be less confusing.

Reply:
Lets do a another PR for this.